app/views/menus/_meal_fields.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="nested-fields" >
+  <%= f.input :food_item_id, collection: FoodItem.all, input_html: {id: "food_item"} %>
```

Comment:
spaces within curly brackets. 

When you relate tags with js, name them like `js-blah-blah`